x-pack/plugin/eql/qa/common/build.gradle

```diff
@@ -9,3 +9,17 @@ dependencies {
   // TOML parser for EqlActionIT tests
   api 'io.ous:jtoml:2.0.0'
 }
+
```

Comment:
I moved some test only dependencies out of the main source set and into the test source set and added expose it as a test artifact.

Reply:
What's the reasoning behind that? If we share code with other projects doing so w/o sharing the entire tests jar is definitely preferable. While "main" is not the best descriport (we should be using the gradle testfixture plugin) it's still better then slamming everything in "test" and then sharing _that_.

Reply:
I can move it back... it just seemed odd to have test only code in the main source set, even for a qa project.

Reply:
Yeah, it's not really "main" code, but its the most convenient place to put it.